src/coreclr/tools/aot/ILCompiler/reproNative/reproNative.vcxproj

```diff
@@ -134,4 +134,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Nope, I reverted the repro proj to HW and it should work as intended

Reply:
I built your branch and reproNative.vcxproj is broken because:

```
1>Runtime.WorkstationGC.lib(startup.cpp.obj) : error LNK2019: unresolved external symbol "void __cdecl EventPipeAdapter_Initialize(void)" (?EventPipeAdapter_Initialize@@YAXXZ) referenced in function "bool __cdecl InitDLL(void *)" (?InitDLL@@YA_NPEAX@Z)
1>Runtime.WorkstationGC.lib(startup.cpp.obj) : error LNK2019: unresolved external symbol "bool __cdecl DiagnosticServerAdapter_Initialize(void)" (?DiagnosticServerAdapter_Initialize@@YA_NXZ) referenced in function "bool __cdecl InitDLL(void *)" (?InitDLL@@YA_NPEAX@Z)
1>Runtime.WorkstationGC.lib(startup.cpp.obj) : error LNK2019: unresolved external symbol "void __cdecl DiagnosticServerAdapter_PauseForDiagnosticsMonitor(void)" (?DiagnosticServerAdapter_PauseForDiagnosticsMonitor@@YAXXZ) referenced in function "bool __cdecl InitDLL(void *)" (?InitDLL@@YA_NPEAX@Z)
1>Runtime.WorkstationGC.lib(startup.cpp.obj) : error LNK2019: unresolved external symbol "void __cdecl EventPipeAdapter_FinishInitialize(void)" (?EventPipeAdapter_FinishInitialize@@YAXXZ) referenced in function "bool __cdecl InitDLL(void *)" (?InitDLL@@YA_NPEAX@Z)
1>Runtime.WorkstationGC.lib(startup.cpp.obj) : error LNK2019: unresolved external symbol "void __cdecl EventPipeAdapter_Shutdown(void)" (?EventPipeAdapter_Shutdown@@YAXXZ) referenced in function "void __cdecl OnProcessExit(void)" (?OnProcessExit@@YAXXZ)
1>Runtime.WorkstationGC.lib(startup.cpp.obj) : error LNK2019: unresolved external symbol "bool __cdecl DiagnosticServerAdapter_Shutdown(void)" (?DiagnosticServerAdapter_Shutdown@@YA_NXZ) referenced in function "void __cdecl OnProcessExit(void)" (?OnProcessExit@@YAXXZ)
1>C:\git\runtime2\src\coreclr\tools\aot\ILCompiler\reproNative\x64\Debug\reproNative.exe : fatal error LNK1120: 6 unresolved externals
```

It needs to be updated to include the (eventpipe-disabled I guess) lib.